app/src/main/res/values-v15/styles.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <resources>
```

Comment:
Actually I doubt if you had test the changes in Android Studio, since I also couldn't get the build to run with my Android Studio after applying this change.

To begin with, remove this redundant `<resources>` tag for the build to pass.

And of course, the closing tag too.